src/datamodel.py

```diff
@@ -154,12 +155,18 @@ def collections_for_key_async(version_key):
 
 class Content(ndb.Model):
   content = ndb.TextProperty(indexed=False)
+  content_compressed = ndb.BlobProperty(indexed=False)
```

Comment:
I think BlobProperty actually has an argument to compress automatically, set `compressed=True`